.yamllint.yaml

```diff
@@ -0,0 +1,8 @@
+---
+extends: default
+
+rules:
+  line-length:
+    ignore: |
+      .github/workflows/*.yml
```

Comment:
I'd remove this line. proper format is 'yaml' and we will stick to it, so 'yml' is not needed here